subsys/net_core_monitor/net_core.c

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/init.h>
+#include <zephyr/types.h>
+#include <errno.h>
+#include <zephyr/kernel.h>
+#include <hal/nrf_reset.h>
+#include "common.h"
+
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(net_core_monitor, CONFIG_NET_CORE_MONITOR_LOG_LEVEL);
+
+#define APP_IPC_GPMEM_0_S  (*(volatile uint32_t *)(0x5002A610))
```

Comment:
I would probably add a short explanation, that IPC address is here because it's not available through standard defines for network core.

Reply:
Done.